nilearn/decoding/decoder.py

```diff
@@ -876,3 +921,301 @@ def __init__(self, estimator='svr', mask=None, cv=10, param_grid=None,
+        e.g. scorer(estimator, X_test, y_test)
+
+        For regression, valid entries are: 'r2', 'neg_mean_absolute_error',
+        or 'neg_mean_squared_error'. Default: 'r2'.
+
+    smoothing_fwhm: float, optional. Default: None
+        If smoothing_fwhm is not None, it gives the size in millimeters of the
+        spatial smoothing to apply to the signal.
+
+    standardize: bool, optional. Default: True
+        If standardize is True, the time-series are centered and normed:
+        their variance is put to 1 in the time dimension.
+
+    target_affine: 3x3 or 4x4 matrix, optional. Default: None
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape: 3-tuple of int, optional. Default: None
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r: float, optional. Default: None
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details.
+
+    mask_strategy: {'background' or 'epi'}, optional. Default: 'background'
+        The strategy used to compute the mask: use 'background' if your
+        images present a clear homogeneous background, and 'epi' if they
+        are raw EPI images. Depending on this value, the mask will be
+        computed from masking.compute_background_mask or
+        masking.compute_epi_mask.
+
+        This parameter will be ignored if a mask image is provided.
+
+    memory: instance of joblib.Memory or str
+        Used to cache the masking process.
+        By default, no caching is done. If a str is given, it is the
+        path to the caching directory.
+
+    memory_level: int, optional. Default: 0
+        Rough estimator of the amount of memory used by caching. Higher value
+        means more memory for caching.
+
+    n_jobs: int, optional. Default: 1.
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs'.
+
+    verbose: int, optional. Default: 0.
+        Verbosity level.
+
+    References
+    ----------
+    .. [1] A. Hoyos-Idrobo, G. Varoquaux, J. Kahn and B. Thirion, "FReM –
+        scalable and stable decoding with fast regularized ensemble of models"
+        in NeuroImage, Elsevier, 2017  pp.1-16, 11 October 2017.
+        https://hal.archives-ouvertes.fr/hal-01615015/
+
+    See Also
+    ------------
+    nilearn.decoding.DecoderRegressor: Regression strategies for Neuroimaging,
+    nilearn.decoding.fREMClassifier: State of the art classification pipeline for Neuroimaging
+    """
+
+    def __init__(self, estimator='svr', mask=None, cv=30, param_grid=None,
```

Comment:
Is there any reason why cv should be set to a quite relatively large number? This would slow the run time a lot I think.

Edit: Also I see both in the example & test file cv is manually set to cv=10.

Reply:
Result from Andres paper in the general case 50 bagged estimators are a good trade-off between performance and computational cost. But for speed in examples and tests I reduced this number since datasets are not full size.